scripts/fast-reboot

```diff
@@ -316,9 +316,6 @@ case "$REBOOT_TYPE" in
             # source mlnx-ffb.sh file with
             # functions to check ISSU upgrade possibility
             source mlnx-ffb.sh
-        elif [[ "$sonic_asic_type" == "barefoot" ]]; then
-            REBOOT_TYPE="fastfast-reboot"
-            BOOT_TYPE_ARG="fastfast"
```

Comment:
the title does not match the code. the code is in the warm-reboot path. why the title says fix fast-reboot for bfn platform?

Reply:
meanwhile, if the intention is the fix the warm reboot path, then that is different from my understand how warm reboot works on barefoot path. My understand of warm reboot on barefoot platform is that you do fast re-init the asic. 

What is the difference between the fast reboot and warm reboot on your platform?

Reply:
@lguohan , thank you for your comments. You are right, its warm-reboot changes. Fixed PR title. The warm-reboot approach on BFN platform has been changed in the latest SDK package https://github.com/Azure/sonic-buildimage/pull/4397 in both master and 201911.